cmd/playground/window_state.go

```diff
@@ -0,0 +1,72 @@
+package playground
+
+import (
+	"time"
+
+	"github.com/odpf/optimus/job"
+)
+
+type state struct {
+	windowV1      job.WindowV1
+	windowV2      job.WindowV2
+	scheduledTime time.Time
+}
+
+func (s *state) incrementScheduleTimeOn(increaseBy cursorPointer) {
```

Comment:
change the name from `increaseBy` into `cursor`

Reply:
this is actually already mentioned in the previous comment. let me know if it's better to be implemented or there's better idea.